test/unit/node/constants.test.ts

```diff
@@ -20,9 +21,14 @@ describe("constants", () => {
     }
 
     beforeAll(() => {
+      jest.clearAllMocks()
```

Comment:
We have `clearAllMocks` in the after callback, do we need it here as well?

Reply:
Hmm...I am thinking out loud. We need one or the other but probably not both 🤔 I think since we do the `clear(testname)` pattern before tests, we should clear the mocks in the `beforeAll` and not the after callback. Thoughts?

Reply:
I like that!

Reply:
Also happy to just leave it as-is for now, not a big deal either way was mostly curious.

Reply:
No no, if I leave it and merge, I won't ever do it lol let's do it right the first time!

Reply:
Consistency is key so keep asking these questions 👏🏼

Reply:
ahahahaha fair fair